themes/grav/app/forms/fields/colorpicker.js

```diff
@@ -82,24 +82,20 @@ export default class ColorpickerField {
         this.wrapper.addClass('cp-visible');
         this.updateFromInput();
 
-        let mainContainer = $('#admin-main .content-wrapper').data('scrollbar').getViewElement();
         this.wrapper.on(MOUSEDOWN, '.cp-grid, .cp-slider, .cp-opacity-slider', this.bound('bodyDown'));
         body.on(MOUSEMOVE, this.bound('bodyMove'));
         body.on(MOUSEDOWN, this.bound('bodyClick'));
         body.on(MOUSEUP, this.bound('targetReset'));
-        $(mainContainer).on('scroll', this.bound('reposition'));
```

Comment:
This is fine removing but you should add it back for the window scroll event. It is meant to auto-reposition the color picker popup when the page scrolls. Above removes completely that functionality.